toc.md

```diff
@@ -165,6 +165,12 @@ docs.vmware.com is built.
         - [Additional resources](authn-authz/additional.md)
             - [Additional resources](authn-authz/additional.md)
             - [Install default roles independently](authn-authz/install.md)
+    - [API Auto Registration](api-auto-registration/about.md)
```

Comment:
Should we update the name of the product to `Tanzu API Auto Registration` to fit into the spot (like our neighbor `Tanzu API portal`), or `API Auto Registration for VMware Tanzu` since that's our official name?

Reply:
I can change it to `API Auto Registration for VMware Tanzu` if that works. @csterwa any opinions?

Reply:
Maybe changing `Tanzu API portal` to `API portal for VMware Tanzu` as well? I don't know how we ended up with that name in the first place, but we can at least keep the names consistent for our two products.